### DISCUSSION TITLE
Events only for object(s) in a zone for x seconds

### DISCUSSION THREAD

**Discussion by spacebares** [blakeblackshear/frigate#5151] [Ask A Question] [answered]
Is it possible for frigate to only create events if an object has been in a zone for x seconds and not just for entering it?

**Comment by NickM-27** [comment, marked as answer]
Not currently but there's a pinned feature request for this.

**Reply by spacebares** [reply]
looks like it might be this https://github.com/blakeblackshear/frigate/issues/2397
